go.mod

```diff
@@ -160,6 +161,7 @@ require (
 
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/cosmos/cosmos-sdk/rosetta => ./server/rosetta
```

Comment:
Once the first release is ready, we should delete this line and link the actual release.
This is done for getting it compile